src/Templates/BlockTemplatesCompatibility.php

```diff
@@ -31,18 +31,6 @@ public function __construct() {
 		$this->init();
 	}
```

Comment:
The `$hook_data` property declaration and `__construct` method of this class should be removed.

Reply:
Fixed via 4c888cf7395c090da1052c8bc2c966a2f6e59c49